.github/workflows/database.yml

```diff
@@ -0,0 +1,42 @@
+name: Database CI
+on: push
+jobs:
+  migrations:
+    name: migrations
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:13
+        env:
+          POSTGRES_PASSWORD: rpg
+          POSTGRES_USER: rpg
+          POSTGRES_DB: rpg
+          DB_PORT: 5432
+        ports:
+          - 5432:5432
```

Comment:
Each job takes various configuration options. 

[`runs-on`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on) determines what kind of computer this runs on (Windows, Ubuntu, Mac, which version, etc). When in doubt, `ubuntu-latest` is the one I go to, since that's usually what servers are running.

For this one, since it's testing the database migrations, we also set up some [services](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices), in particular, the `postgres` service. 

The `image` of the service is the `name:version` of a Docker image from [dockerhub](https://hub.docker.com/search?q=&type=image). I don't know super much about Docker, so I can't give tons of detail on how that works, but basically, Docker lets you run an application in its own isolated virtual machine, and then expose its ports (via the `ports` key) so you can interact with it like it was on a network. The `env` supplies environment variables to configure the service.